packages/protocol/contracts/psp/DaoPspAllocator.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+// This is an example allocator where a DAO admin address can update slots of an Airnode
+// however it wants. What happens in this allocator will not affect what happens in another
+// allocator that the Airnode is using (similar to authorizers).
+// We probably want a more complex DaoPspAllocator where we can have admins of different ranks.
+contract DaoPspAllocator {
+    struct Slot {
+        bytes32 subscriptionId;
+        address sponsor;
+    }
+
+    mapping(address => mapping(uint256 => Slot))
+        public airnodeToSlotIndexToSlot;
+
+    function setSlot(
+        address airnode,
+        uint256 slotIndex,
+        bytes32 subscriptionId,
+        address sponsor
+    ) external {
+        // Check if caller is DAO admin
+        // require(msg.sender == ...)
+        airnodeToSlotIndexToSlot[airnode][slotIndex] = Slot({
```

Comment:
When trying to assign a slot to a new `subscriptionId`, how will we know if the `slotIndex` is not already being used? It feels to me like we are missing a `numberOfSlotsUsed` state var counter that could also be used in `getActiveSubscriptions` by replacing the local var `count`.

Reply:
This `DaoPspAllocator` is just a very heavy-handed implementation that allows the admin to kick subscriptions out at will